src/__test__/components/Header/ProjectInformations.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import ProjectInformations from '@components/Header/ProjectInformations';
+import { BrowserRouter } from 'react-router-dom';
+import Header from '@components/Header';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('Project Informations', () => {
+  test('Test title name',  () => {
+    const { getByText } = render(
+      <BrowserRouter>
+        <ProjectInformations setOpen={jest.fn()}/>
+      </BrowserRouter>
+    );
+    const title = getByText('Atlas de Oportunidades');
+    expect(title).toBeTruthy;
+  });
+
+  test('Test to close button behavior', async () => {
+    const { getByRole } = render( 
+      <BrowserRouter>
+        <Header isComparisonModeOn comparisonType="grid" setComparisonType={() => {}} />
+        <ProjectInformations setOpen={jest.fn()}/>
+      </BrowserRouter>
+    );
+    const MenuButton = await getByRole('MenuButton');
+
+    const closeButton = await getByRole('closeButton');
+
+    fireEvent(
+      MenuButton,
+      new MouseEvent('click', {
+        bubbles: true,
+        cancelable: true,
+      })
+    );
```

Comment:
**suggestion:** take a look at the `userEvent` function, it is even easier to use and not so hard.

https://testing-library.com/docs/ecosystem-user-event/